lib/livebook_web/live/hub/teams/deployment_group_live.ex

```diff
@@ -114,6 +120,36 @@ defmodule LivebookWeb.Hub.Teams.DeploymentGroupLive do
                     return_to={~p"/hub/#{@hub.id}/deployment-groups/edit/#{@deployment_group.id}"}
                   />
                 </div>
+
+                <div :if={@deployment_group.mode == "online"} class="flex flex-col space-y-4">
+                  <h2 class="text-xl text-gray-800 font-medium pb-2 border-b border-gray-200">
+                    Agent Keys
+                  </h2>
+
+                  <p class="text-gray-700">
+                    Deployment group agent keys for Airgapped deployments
```

Comment:
Airgapped are offline :D

```suggestion
                    Deployment group agent keys for online deployments.
```